### PR TITLE
For Merge | Revise the plugin settings page to enhance user-friendliness.

### PR DESCRIPTION
**Pull request by kishanm-wtbx**
For Merge
-----------------
- Included plugin setting page element(Community Address) design changes and copy button related changes.

**Community Address Element design changes**
Non Embed case
image

Embed case
image

Responsive View
image
image

**Note:** Dependent forum branch needs to live. Branch: embed_redirect_issue_23761, PR: https://github.com/webtoolbox/Website-Toolbox/pull/6406

### CODE REVIEW

**Review by masihur — APPROVED** [review]

**Review by webtoolbox — APPROVED** [review]

File: src/resources/websitetoolbox.js [hunk 1]
```diff
@@ -98,9 +98,19 @@ wtbx.setting = {
                     if(subDomainMsg) subDomainMsg.classList.add('hidden');
                     if(forumAddressElement) forumAddressElement.classList.remove("hidden");
                     if(communityWebAddressElement) communityWebAddressElement.classList.add("hidden");
-                } 
+                }
+                wtbx.setting.submitSettignPageForm();
             });
         }
+    }, 
+    submitSettignPageForm: function() {
```
Comment (masihur) [inline src/resources/websitetoolbox.js]: Please correct the spelling the function name should be "submitSettingPageForm"

**Review by masihur — CHANGES_REQUESTED** [review]

**Review by masihur — APPROVED** [review]

**Review by webtoolbox — APPROVED** [review]

File: src/resources/websitetoolbox.js [hunk 2]
```diff
@@ -53,33 +52,34 @@ wtbx.setting = {
             }
         }
         navigator.clipboard.writeText(copyText);
-        element.text = '';
         element.removeAttribute('data-icon');
+        if(window.screen.width >= 575){
```
Comment (masihur) [inline src/resources/websitetoolbox.js]: Please co-ordinate with design team, it should be fix via CSS instead of JS code.

**Review by masihur — CHANGES_REQUESTED** [review]

**Review by masihur — APPROVED** [review]

**Review by webtoolbox — APPROVED** [review]

**Review by masihur — APPROVED** [review]

**Review by webtoolbox — APPROVED** [review]

**Review by masihur — APPROVED** [review]

**Review by webtoolbox — CHANGES_REQUESTED** [review]
Is this index.php URL what you see CraftCMS websites using? Don't they use more visibly-appealing URLs that just call index.php in the backend?

See: https://chatgpt.com/share/670775c0-7c10-800c-ae25-b9a4c765f469

So then the URL you're using in the link and in the text box is not correct.

**Comment by kishanm-wtbx** [comment]
Sir, this index.php is used by Craft CMS. We can remove it from the URL, but I have kept it because our plugin handles both cases. This ensures that if any client’s website URL includes index.php, the plugin will support it as well.

**Comment by webtoolbox** [comment]
This doesn't address what I was saying to you. Yes, the sites use index.php, but that is used in the backend and in the front-end, the URLs the user actually sees are normal URLs. It uses URL rewriting in the web server to map those URLs to index.php. Carefully read and understand everything I sent you.

**Review by masihur — APPROVED** [review]

File: src/templates/settings.twig [hunk 3]
```diff
@@ -179,7 +179,7 @@
                     <a href="{{ settings.forumUrl }}/tool/members/domain?tool=mb" class="change_link mt_5">Change</a> 
                 </div>
                 {{ _self.errorList(settings.getErrors('communityUrl')) }}
-                <span><a id="copyLinkText">Copy the address</a> above and add a link to it in the website’s navigation menu.<a id='frmUrl' href="#" class="d_none">{{ baseUrl }}{{ settings.communityUrl }}</a></span>
+                <span><a id="copyLinkText">Copy the address</a> above and add a link to it in the website’s navigation menu.<a id="frmUrl" href="#" class="d_none">{{ baseUrl }}{% if backwardSlashVisibility == '' %}/{% endif %}{{ settings.communityUrl }}</a></span>
```
Comment (webtoolbox) [inline src/templates/settings.twig:182]: Did you previously define a variable called backwardSlashVisibility?

**Review by webtoolbox — APPROVED** [review]

**Comment by webtoolbox** [comment]
Ensure that there is no visible index.php.

File: src/templates/settings.twig (hunk 3 above)
Comment (kishanm-wtbx) [reply src/templates/settings.twig:182]: Yes

**Comment by kishanm-wtbx** [comment]
> Ensure that there is no visible index.php.

Sir, I will cross-check it. The index.php will appear in the embedded URL if the website URL includes index.php; if the website URL does not include it, then index.php will also not there in the embedded URL.

**Pull request merged**